app/src/main/java/com/cornellappdev/android/eatery/AboutFragment.java

```diff
@@ -0,0 +1,68 @@
+package com.cornellappdev.android.eatery;
+
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v4.app.Fragment;
+import android.text.method.LinkMovementMethod;
+import android.view.LayoutInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ScrollView;
+import android.widget.TextView;
+
+import com.cornellappdev.android.eatery.model.EateryBaseModel;
+
+import java.util.ArrayList;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class AboutFragment extends Fragment {
+
+	private TextView mFeedbackText;
```

Comment:
the indentation in this file (and the others) is 8, can this be changed to 4 everywhere? Make sure your tabs are being converted to spaces properly!